.github/ISSUE_TEMPLATE/cv-x-if-v1.0.0-rc.1-specification-review.yml

```diff
@@ -0,0 +1,41 @@
+name: CV-X-IF v1.0.0-rc.1 Specification Public Review
```

Comment:
i dont think that is needed separately

Reply:
@christian-herber-nxp new version simplifes the spec review template and removes specific version